### ISSUE TITLE
[Feature Request] Allow typing in time to convert timezones

### ISSUE THREAD

**Issue by gowthamravichandran** [n0shake/clocker#124]
Hey @n0shake! Thanks for creating this tool, I have recommended this tool to so many people! I have largely benefitted from this tool! 

Apart from checking current time in different cities, I also use it to convert timezones like 9AM Central is 7:30PM IST. I currently use the time scroller to scroll the time to the exact time and find the equivalent time in other timezones.

### Problem statement
When one needs to scroll like 10 hours, Using the time scroller is difficult and time-consuming to scroll to the exact time one needs to convert. 

### Proposed Solution
Upon clicking the time next to the city name/TZ name, the time can be a editable field, where the user can key in say 9:00 AM, this automatically adjusts equivalent time in other timezones (similar to scroller functionality). 

<img width="300" alt="Screenshot 2023-09-04 at 1 47 41 PM" src="https://github.com/n0shake/clocker/assets/16032220/2eb728bf-12f8-4961-9828-e19a892b1b31">

This would allow the user to convert the time instantly rather than having to fiddle with scroller to get the exact time.

Let me know your thoughts! Open to other ideas as well!

**Comment by eduardo-haro** [comment]
Agree! This tool is amazing, and it helps me a lot. It would be great to have this new feature of typing the time instead of scroll to convert time zones.

**Comment by gowthamravichandran** [comment]
I see #39 closed with a resolution, although not sure how to utilize the feature.
